src/chart/table/TableChart.tsx

```diff
@@ -203,7 +203,7 @@ const NeoTableChart = (props: ChartProps) => {
             navigator.clipboard.writeText(e.value);
           }}
           pageSize={tablePageSize > 0 ? tablePageSize : 5}
-          rowsPerPageOptions={[5]}
+          rowsPerPageOptions={rows.length < 5 ? [rows.length, 5] : [5]}
```

Comment:
Why are we adding this customization here? don't really understand the use case
![image](https://github.com/neo4j-labs/neodash/assets/1754968/02ef9a02-5224-487c-99ac-a07d8a45d226)

Reply:
just removing one console error